src/schema/schema.graphql

```diff
@@ -1400,6 +1400,8 @@ type AccountInfoAttrs {
 	zimbraFeatureImportFolderEnabled: Boolean
 	zimbraFeatureExportFolderEnabled: Boolean
 	zimbraFeatureGroupCalendarEnabled: Boolean
+	zimbraFeatureMailRecallEnabled: Boolean
+	zimbraFeatureMailRecallTime: Int
```

Comment:
what would be maximum value which soap api can give for this property? wanted to make sure we are not running into issues with data which Integer can't hold

Reply:
Maximum value for zimbraFeatureMailRecallTime is 30 min